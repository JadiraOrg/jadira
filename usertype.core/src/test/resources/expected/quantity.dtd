<!ELEMENT dataset (
    QUANTITY*)>

<!ELEMENT QUANTITY EMPTY>
<!ATTLIST QUANTITY
    ID CDATA #REQUIRED
    NAME CDATA #IMPLIED
    QUANTITY CDATA #IMPLIED
>

