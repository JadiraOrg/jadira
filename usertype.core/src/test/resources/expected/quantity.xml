<?xml version='1.0' encoding='UTF-8'?>
<dataset>
  <quantity ID="0" NAME="test_0" QUANTITY="10"/>
  <quantity ID="1" NAME="test_1"/>
</dataset>
